classes/command/class-duplicated-postmeta.php

```diff
@@ -64,7 +64,29 @@ public static function detect() {
 				WHERE meta_key IN (" . self::generate_placeholders( self::META_KEY_LIST, 1 ) . ")
 				GROUP by `post_id`
 				HAVING all_count <> unique_count 
-				ORDER BY `all_count` DESC ";
+				ORDER BY `all_count` DESC";
+
+		$prepared_sql = $wpdb->prepare( $sql, self::META_KEY_LIST );
+
+		return $wpdb->get_results( $prepared_sql );
+		// phpcs:enable
+	}
+
+	/**
+	 * Detect duplicate postmeta meta_key's with count
+	 *
+	 * @return object
+	 */
+	public static function detect_metakey_counts() {
+
+		global $wpdb;
+
+		// phpcs:disable
+		$sql = "SELECT `meta_key`, COUNT(post_id) AS all_count , COUNT(DISTINCT post_id) AS unique_count
+				FROM `wp_postmeta`
+				GROUP by `meta_key`
+				HAVING all_count <> unique_count
+				ORDER BY `all_count` DESC";
 
 		$prepared_sql = $wpdb->prepare( $sql, self::META_KEY_LIST );
```

Comment:
This doesn't have the parameter `self::META_KEY_LIST`

Reply:
ops, my bad. good catch 👍